app/src/main/java/ca/llamabagel/transpo/ui/trips/TripsViewModel.kt

```diff
@@ -81,19 +87,27 @@ class TripsViewModel @Inject constructor(private val tripsRepository: TripsRepos
     }
 
     private fun updateViewerData() {
-        if (selectedRoutes.isEmpty()) return
+        /*if (selectedRoutes.isEmpty()) return
 
         val routeData = (apiData as Result.Success).data.routes
         _viewerData.value = routeData
             .filter { route -> selectedRoutes.contains(RouteSelection(route.number, route.directionId)) }
             .flatMap { route -> route.trips.map { trip -> TripUiModel(route, trip) } }
             .sortedBy { model -> model.trip.adjustedScheduleTime }
-            .map(::TripItem)
+            .map(::TripItem)*/
```

Comment:
Are we still keeping this comment?

Reply:
No. it's been moved to a usecase, will remove